client/components/select-dropdown/index.jsx

```diff
@@ -7,7 +7,7 @@ import find from 'lodash/find';
 import filter from 'lodash/filter';
 import findIndex from 'lodash/findIndex';
 import map from 'lodash/map';
-import result from 'lodash/result';
+import get from 'lodash/get';
```

Comment:
We can combine these into `import { find, filter, findIndex, get, map } from 'lodash';`